### DISCUSSION TITLE
本地打包后访问打包后的html，无法正常加载

### DISCUSSION THREAD

**Discussion by lyj2209914083** [umijs/umi#11752] [Q&A] [answered]
在完全新建的Umi Max项目中，在设置中添加本地打包后的dist作为publicPath，进行打包后访问打包后的html，无法正常访问页面，显示白屏，资源加载中只加载了umi.js和umi.css，其他拆分的js和css均未加载。
publicPath配置截图：
<img width="1118" alt="image" src="https://github.com/umijs/umi/assets/70557762/871eb8f0-67fd-4f49-95f4-0e0e05efe928">
打开html后资源加载情况截图：
<img width="1741" alt="image" src="https://github.com/umijs/umi/assets/70557762/a6fa6df1-217b-433b-8746-8b8dcab2b232">

**Comment by fz6m** [comment, marked as answer]
`publicPath` 这里只能配网址，没法配本地路径，SPA 应用启动产物必须要一种工具来提供服务，在云上一般是 nginx ，在本地可以用 serve 等，但 umi 默认提供了本地模拟启动产物查看的命令：

```bash
  # 构建后，本地启动模拟服务查看产物效果
  # 注意本地无法跨域，因域名不一致，云 cookie 不会生效等，可能需要代理，最终应该在实际运行环境验证效果。
  pnpm max preview
```

**Comment by TrumanDu** [comment]
如何不在容器中可以打开该页面？
